src/quickgui/qgsquickpositionkit.cpp

```diff
@@ -24,10 +24,24 @@
 QgsQuickPositionKit::QgsQuickPositionKit( QObject *parent )
   : QObject( parent )
   , mAccuracy( -1 )
+  , mAccuracyUnits( QgsUnitTypes::toAbbreviatedString( QgsUnitTypes::DistanceMeters ) )
   , mDirection( -1 )
   , mHasPosition( false )
   , mIsSimulated( false )
 {
+
+  mCoordinateTransformer.reset( new QgsQuickCoordinateTransformer() );
+
+  connect( this, &QgsQuickPositionKit::simulatePositionLongLatRadChanged, this, &QgsQuickPositionKit::onSimulatePositionLongLatRadChanged );
+  connect( this, &QgsQuickPositionKit::mapSettingsChanged, this, &QgsQuickPositionKit::setCoordinateTransformMapSettings );
+  if ( mapSettings() )
```

Comment:
this needs to be set in the setMapSettings() function. At this time you do not have mapsettings associated (the order of associating the properties from QML is not precisely set)...